modules/oauth-configuring-token-inactivity-timeout.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-internal-oauth.adoc
+
+[id="oauth-token-inactivity-timeout_{context}"]
+= Configuring token inactivity timeout for the internal OAuth server
+
+You can configure OAuth tokens to expire after a certain amount of time that they have been inactive. By default, there is no token inactivity timeout set.
+
+[NOTE]
+====
+If token inactivity timeout is also configured in your OAuth client, that value overrides the timeout set in the internal OAuth server configuration.
```

Comment:
_Same comment as above_
For consideration:
s/If token inactivity/If the token inactivity/
s/the timeout set in the/the timeout that is set in the/